### DISCUSSION TITLE
What is the relationship between this project and Signal?

### DISCUSSION THREAD

**Discussion by hnarayanan** [oxen-io/session-desktop#2580] [Q&A] [answered]
I ask because this repository was forked from Signal Desktop.

**Comment by KeeJef** [comment, marked as answer]
Session desktop is a code fork of Signal desktop, other than the parts of code which we share (which has become smaller over time) we do not share any relationship. Sorry for the late response

**Reply by hnarayanan** [reply]
Thank you!
